### PR TITLE
Update anchorMC.sh - removing the use of ideal MFT/ITS alignments

### PR DESCRIPTION
**Pull request by catalinristea**
Ciao,

Removed the use of ideal MFT/ITS alignments.
If this is the right time to have it included, please merge.

Thanks and kind regards,
Catalin.

### CODE REVIEW

**Comment by chiarazampolli** [comment]
Thanks @catalinristea !

Should we also stop prefetching any object? Pinging @sawenzel .

**Comment by sawenzel** [comment]
> Thanks @catalinristea !
> 
> Should we also stop prefetching any object? Pinging @sawenzel .

Yes, I think this should be done. Would also reduce the risk of wrong logic between this script and o2dpg. If we wanted ideal alignment for ITS, MFT ... it should probably be enforced inside o2dpg_sim_workflow.

**Comment by benedikt-voelkel** [comment]
Thanks @catalinristea !
I would actually be in favour of removing the code rather that commenting it out. In my opinion we should try to keep especially the production scripts as clean as we can.
(We can always go back in git anyway to get back the states before changes.)

**Comment by catalinristea** [comment]
Ciao Benedikt,

Right, updated.
For what concerns the removal of prefetching, please one of you do it, as I'm afraid to do mistakes, not knowing all the details of this specific functionality.
Thanks!

Yours,
Catalin.

File: MC/run/ANCHOR/anchorMC.sh [hunk 1]
```diff
@@ -212,32 +212,6 @@ fi
...
-
-CCDBOBJECTS="/CTP/Calib/OrbitReset /GLO/Config/GRPMagField/ /GLO/Config/GRPLHCIF /ITS/Calib/DeadMap /ITS/Calib/NoiseMap /ITS/Calib/ClusterDictionary /TPC/Calib/PadGainFull /TPC/Calib/TopologyGain /TPC/Calib/TimeGain /TPC/Calib/PadGainResidual /TPC/Config/FEEPad /TOF/Calib/Diagnostic /TOF/Calib/LHCphase /TOF/Calib/FEELIGHT /TOF/Calib/ChannelCalib /MFT/Calib/DeadMap /MFT/Calib/NoiseMap /MFT/Calib/ClusterDictionary /FV0/Calibration/ChannelTimeOffset"
-
-${O2_ROOT}/bin/o2-ccdb-downloadccdbfile --host http://alice-ccdb.cern.ch/ -p ${CCDBOBJECTS} -d ${ALICEO2_CCDB_LOCALCACHE} --timestamp ${TIMESTAMP}
-if [ ! "$?" == "0" ]; then
-  echo "Problem during CCDB prefetching of ${CCDBOBJECTS}. Exiting."
-  exit 1
-fi
-
-# -- Create aligned geometry using ITS and MFT ideal alignments to avoid overlaps in geant
```
Comment (chiarazampolli) [inline MC/run/ANCHOR/anchorMC.sh:228]: Should we not leave ITS? @noferini

Comment (catalinristea) [reply MC/run/ANCHOR/anchorMC.sh:228]: Ciao,

Something seems to be wrong for the 2 commits done today - I mean "[Merge branch 'AliceO2Group:master' into master](https://github.com/AliceO2Group/O2DPG/pull/1420/commits/d88a5e91b12e3bfe64f6e5239424e5d07842a906)" this one looks odd.

Thanks,
Catalin.

Comment (benedikt-voelkel) [reply MC/run/ANCHOR/anchorMC.sh:228]: I don't quite understand what the issue is. Probably you merged the upstream branch into your development branch. That is not per se a bad thing. And as far as I can see, it does not compromise this PR.
In the end the history of the upstream `master` branch is guaranteed to be linear anyway.
So first of all the diff of this PR is the most important.

**Comment by benedikt-voelkel** [comment]
Are we expecting anything else @catalinristea ?
Otherwise, if you give green light, I think we should go ahead and merge.

File: MC/run/ANCHOR/anchorMC.sh [hunk 2]
```diff
@@ -212,29 +212,16 @@ fi
...
-CCDBOBJECTS_IDEAL_MC="ITS/Calib/Align MFT/Calib/Align"
```
Comment (chiarazampolli) [inline MC/run/ANCHOR/anchorMC.sh:229]: Hello @catalinristea , 
Thanks, but you removed completed CCDBOBJECTS_IDEAL_MC, but it is needed in the lines below...
Chiara

Comment (chiarazampolli) [reply MC/run/ANCHOR/anchorMC.sh:229]: Ah no, it is just moved up in this github view! Apologies.

**Comment by chiarazampolli** [comment]
Seems good to me too!

**Comment by catalinristea** [comment]
Ciao Chiara,

No, nothing to add from my side.

Thanks, Catalin.

**Review by benedikt-voelkel — APPROVED** [review]

**Pull request merged**